experiments/block-level-dedup/makefile

```
@@ -0,0 +1,28 @@
+ALGORITHM = fixed-size
+CHUNK_SIZE = 8192
+FILE = large_filler.csv
+CARGO_RUN = cargo run
+CARGO = cargo 
+DEDUP_BUILD = "../../Oxen/target/debug/dedup"
+
+
+.PHONY: test unpack pack clean clean-test
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)